src/Raven.Client/Util/JavascriptConversionExtensions.cs

```diff
@@ -18,6 +18,8 @@ namespace Raven.Client.Util
 {
     internal class JavascriptConversionExtensions
     {
+        internal const string TransparentIdentifier = "<>h__TransparentIdentifier";
```

Comment:
BTW we already have it defined in: 

https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Client/Documents/Linq/LinqPathProvider.cs#L33

Would be nice to have such const in single location

Reply:
done